integration-tests/tests/billing-projects.js

```diff
@@ -33,7 +33,7 @@ const billingProjectsPage = (testPage, testUrl) => {
 
     selectProject: (billingProjectName, cloudPlatform = GCP) => {
       const text = cloudPlatform === GCP ? `Google Cloud Platform${billingProjectName}` : `Microsoft Azure${billingProjectName}`
-      return noSpinnersAfter(testPage, { action: () => click(testPage, clickable({ text })) })
+      return click(testPage, clickable({ text }))
```

Comment:
The test failure was occurring because when we click on a project name, a spinner is not showing anymore from the spend report tab. That actually seems like a good thing, since the spend report tab is not selected. When the tab actually is selected, I do see the spinner as the Ajax data is loaded.

I can't figure out why your code change resulted in this change in behavior, but I think we are OK to just change the test so it no longer expects the spinner to show when the project is selected. I ran the test 100 times with "flakes", and didn't encounter any failures.

Even if a spinner does show when the project is selected, the tests should be fine as long as they don't ever assume that data is immediately present. I believe this is case because the tests are always waiting for some text element or control to appear, and it's OK if that takes some time. There was probably no good reason for the test to ever wait for the spinner at that point.